Tribler/Core/APIImplementation/LaunchManyCore.py

```diff
@@ -235,6 +234,10 @@ def load_communities():
         if self.session.get_libtorrent():
             from Tribler.Core.Libtorrent.LibtorrentMgr import LibtorrentMgr
             self.ltmgr = LibtorrentMgr(self.session)
+            self.ltmgr.initialize()
+            # FIXME(lipu): upnp APIs are not exported in libtorrent python-binding.
```

Comment:
The code is there, which is the line after this `FIXME`. I commented it for the moment. This `FIXME` is just a simple documentation.